### ISSUE TITLE
Including germ.duration in the analysis

### ISSUE THREAD

**Issue by wangxm-forest** [lizzieinvancouver/egret#35]
Thanks @kengi-neer for mentioning that not all the studies have the final per.germ, and we might want to include germ.duration in our analysis.

**Comment by lizzieinvancouver** [comment]
@kengi-neer @wangxm-forest Good point! We should do some plotting of this at some point when we're further along with analyses and decide what to do.

**Comment by lizzieinvancouver** [comment]
Thanks @kengi-neer ! We currently have this in the analysis for seedCues by @vvandermeersch
